src/NServiceBus.Core/Recoverability/Settings/RecoverabilitySettings.cs

```diff
@@ -55,6 +56,22 @@ public RecoverabilitySettings CustomPolicy(Func<RecoverabilityConfig, ErrorConte
         }
 
         /// <summary>
+        /// Configures the specified exception type to be treated as an unrecoverable exception.
+        /// </summary>
+        /// <param name="exceptionType">The exception type.</param>
+        /// <remarks>The method can be called multiple times.</remarks>
+        public RecoverabilitySettings UnrecoverableException(Type exceptionType)
```

Comment:
Should we prefix with "Register"/"Add" or something to make it more clear what this is doing?